src/cljc/matcher_combinators/matchers.cljc

```diff
@@ -66,13 +66,14 @@
   (core/->Regex expected))
 
 (def absent
-  "Value-position matcher for maps that matches when containing map doesn't have the key pointing to this matcher."
+  "Value-position matcher for maps that matches when containing map doesn't
+  have the key pointing to this matcher."
   (core/->Absent))
 
 (defn pred
   "Matcher that will match when `pred` of the actual value returns true."
   [pred]
-  (core/->PredMatcher pred (str "predicate: " pred)))
+  (core/->PredMatcher pred `(~'pred ~pred)))
```

Comment:
sneaking this little improvement in as well:

before this change 
```
    (is (match? even? 1)))
```
would print
```
FAIL in (absent-matcher) (matchers_test.clj:384)
assert an entry is definitely not in a sequence
expected: (match? even? 1)
  actual: (mismatch "clojure.core$even_QMARK_@9d37f5c" 1)
```
and after it prints
```
FAIL in (absent-matcher) (matchers_test.clj:384)
assert an entry is definitely not in a sequence
expected: (match? even? 1)
  actual: (mismatch (pred #function[clojure.core/even?]) 1)
```